src/api/app/assets/javascripts/webui/comment.js

```diff
@@ -45,13 +45,13 @@ $(document).ready(function(){
     });
   });
 
-  $('button[id*="edit_button_of_"]').on('click', (e) => {
+  $('body').on('click', 'button[id*="edit_button_of_"]', (e) => {
```

Comment:
Regarding a performance conversation that I had with Daniel (Sorry but I don't know which of the Daniels to annotate here), I want to mention that this event listener will be attached to the `body` tag so the event listeners will not increase as the comment count increases.